pkg/operation/care/health.go

```diff
@@ -345,6 +346,11 @@ func (h *Health) checkSystemComponents(
 		managedResources = append(managedResources, vpnshoot.ManagedResourceName)
 	}
 
+	if versionConstraintGreaterEqual138.Check(checker.gardenerVersion) && h.shoot.NodeLocalDNSEnabled {
```

Comment:
I don't know whether this PR will make it into the `v1.38` release. Maybe change this to `versionConstraintGreaterEqual139` to be safe.